test/scripts/test_bytecodecompare_prepare_report.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 from textwrap import dedent
```

Comment:
This can be removed now, right?

Reply:
Right. Good catch. I wonder why it passed our pylint checks. Looks like I need to take a look at `pylintrc`.